classroom.rb

```diff
@@ -1,16 +1,13 @@
-class Classroom 
-
-    def initialize
-        @label 
-        @students = []
-    end
-
-    attr_accessor :label
-
-    def add_student(student)
-        @students.push(student)
-        student.classroom = self
-    end
-
-
+class Classroom
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  attr_accessor :label
```

Comment:
- [ ] Great job so far 👍 kindly add the `attr_reader :students` to the Classroom class so that a user can check the students of a classroom